reflect/src/main/java/dagger/reflect/ReflectiveMembersInjector.java

```diff
@@ -38,8 +39,7 @@
     Deque<ClassBindings> hierarchyBindings = new ArrayDeque<>();
```

Comment:
```suggestion
     Deque<ClassProviders> hierarchyProviders = new ArrayDeque<>();
```
to be consistent with field|methodProviders?

Reply:
This object isn't queried so I don't think naming it like a collection of something makes sense. It's a binder, maybe. I don't know. Can be figured out in a follow-up.